tests/test_keybind.py

```diff
@@ -221,6 +221,7 @@ def test_keymask_applied_to_new_binds(hlwm, keyboard):
 def test_keymask_prefix(hlwm, keyboard):
     hlwm.call('keybind space set_attr clients.focus.my_space_pressed pressed')
     hlwm.create_client()
+    keyboard.wait_until_window_is_focused(hlwm.attr.clients.focus.winid())
```

Comment:
clients.focus is the window that hlwm wants to have the focus. But the assignment of the input focus is not immediate because hlwm just tells the client that the client should request the input focus from X. The statement waits for X11's input focus to match `clients.focus`.

Reply:
I see, that makes sense, thank you!